packages/paste-website/src/pages/paste-libraries/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Paste Libraries
+description: Standalone tools and plugins built and supported by the Paste team.
+---
+
+import {graphql} from 'gatsby';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {Grid} from '../../components/grid';
+import { Link } from "gatsby"
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: { slug: { eq: "/paste-libraries/" } }) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+---
+
+## Motivation
+
+Paste is built by just a few people, but the reach of our libraries are great.  One of the team's 
+charters is creating a culture of prototypical design and development. While we work on building 
+out the design system, certain challenges we face are best solved by standalone tooling, 
+complimentary to and re-useable outside the framework of a design system.
```

Comment:
Is this section needed? Can we just get to "These libraries are important to our DSYS efforts, etc."